src/Parallel/ParallelComponentHelpers.hpp

```diff
@@ -151,19 +151,34 @@ using get_initialization_tags_to_keep =
         detail::get_initialization_tags_to_keep_from_action<tmpl::_1>>>>;
 
 namespace detail {
-template <typename InitializationTag>
-struct get_option_tags_from_initialization_tag {
+template <typename InitializationTag,typename Metavariables,
+          typename = std::nullptr_t>
```

Comment:
make this last parameter a `bool PassMetavariables = InitializationTag::pass_metavariables`